lib/routes/dev/developers_view.dart

```diff
@@ -109,4 +114,26 @@ class DevelopersView extends StatelessWidget {
     final zipFile = XFile(zipFilePath);
     Share.shareXFiles([zipFile]);
   }
+
+  void _exportStaticBackup(BuildContext context) async {
+    final accBloc = context.read<AccountBloc>();
+    const name = "emergency.recover";
```

Comment:
Is this the name cln use?

Reply:
Yes
https://github.com/ElementsProject/lightning/blob/df10c62508aecb7aaee0fdc4166face87663f8b9/plugins/chanbackup.c#L26

Reply:
I see but this, as implemented right now, is confusing because recovering from emergency file is different than recovering from the gPRc interface.
Here is how we should recover from the static backup hex data: https://docs.corelightning.org/reference/lightning-recoverchannel

If we call it emergency.recover the user would think the right way to recover is this: https://docs.corelightning.org/reference/lightning-emergencyrecover

Reply:
Ah, I didn't find that recoverchannel function.

Reply:
Renamed the file to scb.recover and also return the value as is.